dashboards-observability/public/components/event_analytics/explorer/visualizations/index.tsx

```diff
@@ -73,7 +73,11 @@ export const ExplorerVisualizations = ({
                 handleOverrideTimestamp={handleOverrideTimestamp}
                 handleAddField={(field: IField) => handleAddField(field)}
                 handleRemoveField={(field: IField) => handleRemoveField(field)}
-                isFieldToggleButtonDisabled={true}
+                isFieldToggleButtonDisabled={
```

Comment:
What's the end goal we are trying to achieve here? disable toggle buttons when it's log view and there's no viz data? Don't depend on stats keyword in query as this will change soon.

Reply:
I have changed the logic. My intention was to only enable the Sidebar's field toggle button when the visualization is Logs-View and used the same logic (enabled/disabled) as used in Events tab.

Reply:
Ok got it, then how about change it to something like
`vis.name !== visChartTypes.LogsView` as I recall that visualization sidebar by default disable all fields.

Reply:
Yes, along with that I have added this logic which is used in Events Tab: `isEmpty(explorerData.jsonData) || !isEmpty(query[RAW_QUERY].match(PPL_STATS_REGEX))`. Please let me know whether we should keep the next part or we should remove it.